02-sql-aggregation.md

```diff
@@ -114,32 +114,121 @@ think of views is as a table, that can read, aggregate, and filter information
 from several places before showing it to you.
 
 Creating a view from a query requires to add `CREATE VIEW viewname AS`
-before the query itself. For example, if we want to save the query giving
-the number of individuals in a view, we can write
+before the query itself. For example, imagine that my project only covers
+the data gathered during the summer (May - September) of 2000.  That 
+query would look like: 
 
-    CREATE VIEW species_count AS
-    SELECT species_id, COUNT(*)
+    SELECT *
     FROM surveys
-    GROUP BY species_id;
+    WHERE year == 2000 AND (month > 4 AND month < 10)
```

Comment:
Minor note: "==" is not standard SQL (and is rejected by a majority of engines, although SQLite permits it). A plain "=" is standard (universally accepted).